app/models/user.rb

```diff
@@ -0,0 +1,50 @@
+class User < ApplicationRecord
+  has_secure_password
+
+  VALID_EMAIL_REGEX = Regexp.new(Settings.development.user.valid_email_regex,
+                                 Regexp::IGNORECASE)
+  PASSWORD_REQUIREMENT =
+    Regexp.new(Settings.development.user.password_requirement)
+
+  validates :name, presence: true,
+                  length: {maximum: Settings.development.user
+                                            .max_name_length.to_i}
+
+  validates :email, presence: true, format: {with: VALID_EMAIL_REGEX},
+                    uniqueness: {case_sensitive: false},
+                    length: {maximum: Settings.development.user
+                                              .max_email_length.to_i}
```

Comment:
E add i18n messages custom cho các validates này nha

Reply:
Giữa các thành phần giống nhau trong modal thì k cần khoảng trống đâu nha (ngoại trừ method)
```suggestion
  validates :name, ...
  validates :email, ...
```

Reply:
<img width="1029" height="179" alt="image" src="https://github.com/user-attachments/assets/28769f26-ec86-4f81-aff2-e5844f6a2875" />
 
<img width="587" height="303" alt="image" src="https://github.com/user-attachments/assets/341964b3-9510-436d-8091-414e843030c7" />